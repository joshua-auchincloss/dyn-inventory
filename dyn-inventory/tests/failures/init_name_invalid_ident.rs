// invalid identifier in struct position (hyphen) for registry struct
//
//
//
//
//
//
//
//
//
use dyn_inventory::dyn_inventory;

dyn_inventory!(
    My-Handle<T: Plugin> {
        active: bool,
        t: T,
    };
);

fn main() {}
