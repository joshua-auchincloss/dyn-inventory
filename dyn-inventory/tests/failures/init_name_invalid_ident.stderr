error: expected `<`
  --> tests/failures/init_name_invalid_ident.rs:14:7
   |
14 |     My-Handle<T: Plugin> {
   |       ^
